cmd/opdoc/opdoc.go

```diff
@@ -222,7 +262,7 @@ func opToMarkdown(out io.Writer, op *logic.OpSpec, groupDocWritten map[string]bo
 	for i := range op.OpDetails.Immediates {
 		group := op.OpDetails.Immediates[i].Group
 		if group != nil && group.Doc != "" && !groupDocWritten[group.Name] {
-			fmt.Fprintf(out, "\n`%s` %s:\n\n", group.Name, group.Doc)
+			fmt.Fprintf(out, "\n### Field Group %s\n\n%s\n\n", group.Name, group.Doc)
```

Comment:
I'm not sure sure about saying "Field Group" here.  That is what we call them internally, but only because the grew out of the `txn` opcode, where they really are "fields" of the transaction. In other contexts, "field" seems wrong.

Reply:
Suggestions?

Immediate Enum? Argument Enum?

Reply:
My thought was just use `group.Name` and `group.Doc` Those words were chosen so this sort of thing would work, like `"ECDSA", "Curves"`.
Is there one (or more) that reads poorly?